filebeat/inputsource/common/streaming/listener.go

```diff
@@ -80,14 +79,6 @@ func (l *Listener) Start() error {
 		return err
 	}
 
-	l.ctx, l.cancel = context.WithCancel(context.Background())
-	go func() {
-		<-l.ctx.Done()
-		l.Listener.Close()
-	}()
-
-	l.log.Info("Started listening for " + l.family.String() + " connection")
```

Comment:
This code-block should not have existed in 7.x. Maybe some older backport that didn't go well.

Removing this block fixes:
- reduce chance for leaking resources due to an existing context being overwritten
- remove duplicate log message.